src/main/java/com/deavensoft/timetracker/endpoint/ImportExcelEndpoint.java

```diff
@@ -0,0 +1,63 @@
+package com.deavensoft.timetracker.endpoint;
+
+import com.deavensoft.timetracker.api.model.request.JiraMappingRequest;
+import com.deavensoft.timetracker.domain.jira.JiraProject;
+import com.deavensoft.timetracker.domain.jira.JiraUser;
+import com.deavensoft.timetracker.service.ImporterService;
+import com.deavensoft.timetracker.service.jira.JiraProjectService;
+import com.deavensoft.timetracker.service.jira.JiraUserService;
+import java.io.IOException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+@RequestMapping(ImportExcelEndpoint.BASE_URL)
+@RequiredArgsConstructor
+@CrossOrigin(maxAge = 3600)
+@RestController
+@Slf4j
+public class ImportExcelEndpoint {
+
+  public static final String BASE_URL = "/v1.0/excel";
```

Comment:
Please, add for all JIRA related stuff, endpoints should have a prefix: `/v1.0/jira/`